tests/compile/test_rocm_aiter_fusion.py

```diff
@@ -0,0 +1,257 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: Copyright contributors to the vLLM project
+from importlib.util import find_spec
+
+import pytest
+import torch
+
+from vllm._aiter_ops import rocm_aiter_ops
+from vllm.compilation.activation_quant_fusion import SILU_MUL_OP
+from vllm.compilation.noop_elimination import NoOpEliminationPass
+from vllm.compilation.post_cleanup import PostCleanupPass
+from vllm.compilation.rocm_aiter_fusion import (
+    RocmAiterRMSNormFp8GroupQuantFusionPass,
+    RocmAiterSiluMulFp8GroupQuantFusionPass,
+)
+from vllm.config import (
+    CompilationConfig,
+    CompilationMode,
+    ModelConfig,
+    PassConfig,
+    VllmConfig,
+    set_current_vllm_config,
+)
+from vllm.model_executor.layers.activation import SiluAndMul
+from vllm.model_executor.layers.quantization.utils.fp8_utils import W8A8BlockFp8LinearOp
+from vllm.model_executor.layers.quantization.utils.quant_utils import GroupShape
+from vllm.model_executor.layers.quantization.utils.w8a8_utils import (
+    maybe_create_device_identity,
+)
+from vllm.platforms import current_platform
+
+from .backend import TestBackend
+
+FP8_DTYPE = current_platform.fp8_dtype()
+
+# Check if aiter package is installed
+aiter_available = find_spec("aiter") is not None
+
+
+class TestAiterSiluMulGroupFp8QuantModel(torch.nn.Module):
```

Comment:
Tests would be more maintainable if we combine with existing fusion tests, is there a limit to that? See #27883

Reply:
@tjtanaa can you elaborate? I just worry that these tests are already very verbose and there's a lot of them so if we add more they will be harder to maintain, people won't clean them up, and they're more likely to break

Reply:
I totally get that the patterns have to be different, but the tests should be the same? And just use `W8A8LinearOp` which will produce the aiter or "regular" forward code either way?